api/src/main/java/ca/bc/gov/educ/api/ruleengine/util/JsonTransformer.java

```diff
@@ -0,0 +1,103 @@
+package ca.bc.gov.educ.api.ruleengine.util;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Component
+public class JsonTransformer implements Transformer {
```

Comment:
No need for this class if we are removing logClientHttpReqResponseDetails in LogHelper Class